modules/RTCStats/RTCStats.ts

```diff
@@ -148,6 +197,8 @@ class RTCStats {
             }
 
             this.sendIdentity(identityData);
+            // Reset the flag, so that the next conference that is joined will have the trace module initialized, such as a breakout room.
+            this._attachedAtConnection = false;
```

Comment:
I can't come up with one right now, but this name reads weird. I see what you want to accomplish, but technically you were attached already so...

Reply:
changed the name to be more suggestive.